guides/common/modules/proc_upgrading-project-in-place-using-leapp.adoc

```diff
@@ -50,6 +58,39 @@ On {RHEL}, enable the `rhel-7-server-extras-rpms` repository:
 ----
 # {package-install-project} leapp leapp-repository
 ----
+ifdef::satellite[]
+. For Leapp to perform the upgrade in a disconnected environment, the metadata should be downloaded and extracted manually, as described in https://access.redhat.com/articles/3664871[Leapp utility metadata in-place upgrades of RHEL for disconnected upgrades].
```

Comment:
Please reword "should be downloaded".

Maybe "Download the ....
For more information see X." ?